### DISCUSSION TITLE
Publish releases on GitHub as well

### DISCUSSION THREAD

**Discussion by alroniks** [modxcms/revolution#15690] [General]
Hi. I saw that we usually when we releasing a new version just by creating a tag, but we do not use the Releases feature on GitHub at all.

For an open-source project, it looks ridiculous a bit, and even more, the releases section on the main page of the repo is empty. It is not good.

I suggest publishing releases as well on Github, including valuable changelogs and archives with different versions in the assets. The same as on the website, but also o GitHub.

I prefer to automate such things, but the first time I can volunteer and handle them as my small responsibility.

What do you think, guys? @rthrash @opengeek @jaygilmore

**Comment by opengeek** [comment]
I'm a 2/10 on this. It's just going to increase the work needed to release. That said, if we could work to remove the need for a Phing build completely, I think it would make much more sense. But, I'm also exploring adding the Phing dependency and a script to do the build via Composer. IMO, I would really like to focus on getting 3.0.0 out the door, and then we can work on things like this which will require altering some processes.
